code/_core/turf/simulated/floor/astroturf.dm

```diff
@@ -3,5 +3,7 @@
 	name = "astroturf"
 	icon = 'icons/turf/floor/natural.dmi'
 	icon_state = "grass"
+	desc = "Fake grass"
+	var/desc_extended = "it doesn't feel pain when you step on it, unlike real grass"
```

Comment:
``var/desc_extended`` caused compile errors because it's already defined. It should be just ``desc_extended``

Reply:
ah whoops, went over the code real quick and removed those